optaplanner-examples/src/main/java/org/optaplanner/examples/meetingscheduling/domain/Meeting.java

```diff
@@ -26,7 +26,7 @@
 
     private String topic;
     /**
-     * Multiply by {@link TimeGrain#GRAIN_LENGTH_IN_MINUTES} to get duration in minutes
+     * Multiply by {@link TimeGrain#GRAIN_LENGTH_IN_MINUTES} to get duration in minutes.
```

Comment:
Nitpick: This is technically not a sentence, because it has no verb (javadoc style says sentence parts should not end in dot IIRC). Not important, so I 'll just merge anyway.